python/lancedb/embeddings/openai.py

```diff
@@ -51,4 +53,7 @@ def generate_embeddings(
     @cached_property
     def _openai_client(self):
         openai = self.safe_import("openai")
+
+        if not os.environ.get("OPENAI_API_KEY"):
+            raise ValueError(api_key_not_found_help("openai"))
```

Comment:
api_key_not_found_help should raise an error internally, you don't need to raise it here I think

Reply:
Yes, indeed, it can be like this:
```
if not os.environ.get("OPENAI_API_KEY"):
     api_key_not_found_help("openai")
```

Initially, I used this [line](https://github.com/lancedb/lancedb/blob/main/python/lancedb/embeddings/gemini_text.py#L127) as a template. I can fix this in two places.

Reply:
Thanks!